From 1bb92d8b281297547a612475353e3871f875cb8c Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Wed, 4 Mar 2020 15:52:13 +0800
Subject: [PATCH] drm/imx: lcdif: fix Parameter passing error when calling
 ftomic_helper_check_plane_state function

Use drm_atomic_helper_check_plane_state The second parameter needs to
be changed to crtc_state because the parameter type has been modified

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 drivers/gpu/drm/imx/lcdif/lcdif-plane.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/lcdif/lcdif-plane.c b/drivers/gpu/drm/imx/lcdif/lcdif-plane.c
index 587d257ab5dd..e329cae28529 100644
--- a/drivers/gpu/drm/imx/lcdif/lcdif-plane.c
+++ b/drivers/gpu/drm/imx/lcdif/lcdif-plane.c
@@ -71,7 +71,7 @@ static int lcdif_plane_atomic_check(struct drm_plane *plane,
 	clip.x2 = mode->hdisplay;
 	clip.y2 = mode->vdisplay;
 
-	ret = drm_atomic_helper_check_plane_state(plane_state, &clip,
+	ret = drm_atomic_helper_check_plane_state(plane_state, crtc_state,
 					   DRM_PLANE_HELPER_NO_SCALING,
 					   DRM_PLANE_HELPER_NO_SCALING,
 					   false, true);
-- 
2.17.1

